docs/UserGuide.adoc

```diff
@@ -54,66 +54,62 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 * Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 ====
 
-=== Viewing help : `help`
-
-Format: `help`
-
-
-=== Clearing all entries : `avail`
-
-Lists all the available rooms of that type for the next NUM_DAYS days +
-Format: `avail r/ROOM TYPE num/NUM_DAYS`
-
-Lists all available rooms from start to end date. Dates have to be given in YYMMDD format
-
-=== Adding a guest: `add`
+=== Adding a person: `add`
 
```

Comment:
Note: this section has to be updated by the person working on the add command (Joyce)